src/lib/sql/sql_translator.cpp

```diff
@@ -475,7 +475,7 @@ SQLTranslator::TableSourceState SQLTranslator::_translate_table_origin(const hsq
         }
 
         AssertInput(_use_mvcc == (lqp_is_validated(view->lqp) ? UseMvcc::Yes : UseMvcc::No),
-               "Mismatch between validation of View and query it is used in");
+                    "Mismatch between validation of View and query it is used in");
```

Comment:
Random line: This doesn't test zero-suppression yet, does it?

Reply:
You mean this VectorCompressionType thing? Are all EncodingTypes (except unencoded) compatible with all VectorCompressionTypes? If not, how do I figure out which are compatible?

Reply:
I think right now it's only stored in `encoded_segment_test.cpp`. We probably should have that somewhere in the actual code base, not only in the tests.

Reply:
I left a note about this in the original issue. Once the blockers for running the SQL tests with encoding are fixed and encoding is activated for the SQL tests, we should active VectorCompression as well.

Instantiation will then probably look something like this:

```
INSTANTIATE_TEST_CASE_P(SQLiteTestRunnerEncodings, SQLiteTestRunner,
                        testing::Combine(testing::ValuesIn(SQLiteTestRunner::queries()), testing::ValuesIn({false}),
                                         testing::ValuesIn({{EncodingType::Dictionary, VectorCompressionType::Invalid},
                                                                      {EncodingType::Dictionary, VectorCompressionType::SimdBp128}, ...)), );  // NOLINT
```